Feed loader failing to install on 5.0.0-beta.2

Hi, I'm excited to take the feed loader for a spin but failing to install it. My deps are

```json
    "@ascorbic/feed-loader": "^1.0.1",
    "astro": "^5.0.0-beta.2"
```

```sh
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: test.astro@0.0.1
npm error Found: astro@5.0.0-beta.2
npm error node_modules/astro
npm error   astro@"^5.0.0-beta.2" from the root project
npm error
npm error Could not resolve dependency:
npm error peer astro@"^4.14.0 || ^5.0.0" from @ascorbic/feed-loader@1.0.1
npm error node_modules/@ascorbic/feed-loader
npm error   @ascorbic/feed-loader@"^1.0.1" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
```
If I run `npm i --legacy-peer-deps`  I get

```sh
Cannot find package 'typescript' imported from /Users/test.astro/node_modules/zod-to-ts/dist/index.js
  Stack trace:
    at new NodeError (node:internal/errors:405:5)
    at moduleResolve (node:internal/modules/esm/resolve:973:20)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:404:12)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:250:38)
    at link (node:internal/modules/esm/module_job:75:36)
```

--- a/.changeset/eight-fans-judge.md
+++ b/.changeset/eight-fans-judge.md
@@ -1,0 +1,9 @@
+---
+"@ascorbic/airtable-loader": patch
+"@ascorbic/loader-utils": patch
+"@ascorbic/feed-loader": patch
+"@ascorbic/mock-loader": patch
+"@ascorbic/csv-loader": patch
+---
+
+Fix peer dependencies for Astro 5 beta

--- a/.gitignore
+++ b/.gitignore
@@ -28,3 +28,4 @@ dist
 .netlify
 
 *.tgz
+.DS_Store

--- a/packages/airtable/package.json
+++ b/packages/airtable/package.json
@@ -19,13 +19,13 @@
 	},
 	"devDependencies": {
 		"@arethetypeswrong/cli": "^0.16.4",
-		"astro": "^4.16.6",
+		"astro": "5.0.0-beta.5",
 		"publint": "^0.2.11",
 		"tsup": "^8.3.0",
 		"typescript": "^5.6.3"
 	},
 	"peerDependencies": {
-		"astro": "^4.14.0 || ^5.0.0"
+		"astro": "^4.14.0 || ^5.0.0-beta.0"
 	},
 	"keywords": [
 		"astro",

--- a/packages/csv/package.json
+++ b/packages/csv/package.json
@@ -20,7 +20,7 @@
 	"devDependencies": {
 		"@arethetypeswrong/cli": "^0.16.4",
 		"@types/papaparse": "^5.3.14",
-		"astro": "^4.16.6",
+		"astro": "5.0.0-beta.5",
 		"publint": "^0.2.11",
 		"tsup": "^8.3.0",
 		"typescript": "^5.6.3"

--- a/packages/feed/package.json
+++ b/packages/feed/package.json
@@ -20,13 +20,13 @@
 	"devDependencies": {
 		"@arethetypeswrong/cli": "^0.16.4",
 		"@types/feedparser": "^2.2.8",
-		"astro": "^4.16.6",
+		"astro": "5.0.0-beta.5",
 		"publint": "^0.2.11",
 		"tsup": "^8.3.0",
 		"typescript": "^5.6.3"
 	},
 	"peerDependencies": {
-		"astro": "^4.14.0 || ^5.0.0"
+		"astro": "^4.14.0 || ^5.0.0-beta.0"
 	},
 	"keywords": [
 		"astro",

--- a/packages/mock/package.json
+++ b/packages/mock/package.json
@@ -19,13 +19,13 @@
 	},
 	"devDependencies": {
 		"@arethetypeswrong/cli": "^0.16.4",
-		"astro": "^4.16.6",
+		"astro": "5.0.0-beta.5",
 		"publint": "^0.2.11",
 		"tsup": "^8.3.0",
 		"typescript": "^5.6.3"
 	},
 	"peerDependencies": {
-		"astro": "^4.14.0 || ^5.0.0"
+		"astro": "^4.14.0 || ^5.0.0-beta.0"
 	},
 	"keywords": [
 		"astro",

--- a/packages/utils/package.json
+++ b/packages/utils/package.json
@@ -19,13 +19,13 @@
 	},
 	"devDependencies": {
 		"@arethetypeswrong/cli": "^0.16.4",
-		"astro": "^4.16.6",
+		"astro": "5.0.0-beta.5",
 		"publint": "^0.2.11",
 		"tsup": "^8.3.0",
 		"typescript": "^5.6.3"
 	},
 	"peerDependencies": {
-		"astro": "^4.14.0 || ^5.0.0"
+		"astro": "^4.14.0 || ^5.0.0-beta.0"
 	},
 	"keywords": [],
 	"author": "",

